weblate/trans/models/change.py

```diff
@@ -296,109 +296,109 @@ class Change(models.Model, UserDisplayMixin):
         # Translators: Name of event in the history
         (ACTION_SUGGESTION_DELETE, gettext_lazy("Suggestion removed")),
         # Translators: Name of event in the history
-        (ACTION_REPLACE, gettext_lazy("Search and replace")),
+        (ACTION_REPLACE, gettext_lazy("Translation replaced")),
         # Translators: Name of event in the history
         (ACTION_SUGGESTION_CLEANUP, gettext_lazy("Suggestion removed during cleanup")),
         # Translators: Name of event in the history
         (ACTION_SOURCE_CHANGE, gettext_lazy("Source string changed")),
         # Translators: Name of event in the history
-        (ACTION_NEW_UNIT, gettext_lazy("New string added")),
+        (ACTION_NEW_UNIT, gettext_lazy("String added")),
         # Translators: Name of event in the history
-        (ACTION_BULK_EDIT, gettext_lazy("Bulk status change")),
+        (ACTION_BULK_EDIT, gettext_lazy("Bulk status changed")),
         # Translators: Name of event in the history
-        (ACTION_ACCESS_EDIT, gettext_lazy("Changed visibility")),
+        (ACTION_ACCESS_EDIT, gettext_lazy("Visibility changed")),
         # Translators: Name of event in the history
-        (ACTION_ADD_USER, gettext_lazy("Added user")),
+        (ACTION_ADD_USER, gettext_lazy("User added")),
         # Translators: Name of event in the history
-        (ACTION_REMOVE_USER, gettext_lazy("Removed user")),
+        (ACTION_REMOVE_USER, gettext_lazy("User removed")),
         # Translators: Name of event in the history
         (ACTION_APPROVE, gettext_lazy("Translation approved")),
         # Translators: Name of event in the history
         (ACTION_MARKED_EDIT, gettext_lazy("Marked for edit")),
         # Translators: Name of event in the history
-        (ACTION_REMOVE_COMPONENT, gettext_lazy("Removed component")),
+        (ACTION_REMOVE_COMPONENT, gettext_lazy("Component removed")),
         # Translators: Name of event in the history
-        (ACTION_REMOVE_PROJECT, gettext_lazy("Removed project")),
+        (ACTION_REMOVE_PROJECT, gettext_lazy("Project removed")),
         # Translators: Name of event in the history
-        (ACTION_RENAME_PROJECT, gettext_lazy("Renamed project")),
+        (ACTION_RENAME_PROJECT, gettext_lazy("Project renamed")),
         # Translators: Name of event in the history
-        (ACTION_RENAME_COMPONENT, gettext_lazy("Renamed component")),
+        (ACTION_RENAME_COMPONENT, gettext_lazy("Component renamed")),
         # Translators: Name of event in the history
-        (ACTION_MOVE_COMPONENT, gettext_lazy("Moved component")),
+        (ACTION_MOVE_COMPONENT, gettext_lazy("Component moved")),
         # Using pgettext to differentiate from the plural
         # Translators: Name of event in the history
         (
             ACTION_NEW_STRING,
-            pgettext_lazy("Name of event in the history", "New string to translate"),
+            pgettext_lazy("Name of event in the history", "String added"),
```

Comment:
There are two `String added` after changing `New string added` to `String added`. It is needed to distinguish these two events.

Reply:
Good point, I think it's time to drop this event. Tracked separately at https://github.com/WeblateOrg/weblate/issues/10153